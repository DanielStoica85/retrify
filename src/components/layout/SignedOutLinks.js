import React from 'react';
import { NavLink, NavItem } from 'reactstrap';

const SignedOutLinks = () => (
    <React.Fragment>
        <NavItem>
            <NavLink href="/register">Sign Up</NavLink>
        </NavItem>
        <NavItem>
            <NavLink href="/login">Login</NavLink>
        </NavItem>
    </React.Fragment>
);

export default SignedOutLinks;
