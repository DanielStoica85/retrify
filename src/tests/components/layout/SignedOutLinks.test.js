import React from 'react';
import { shallow } from 'enzyme';
import SignedOutLinks from '../../../components/layout/SignedOutLinks';

describe('SignedInLinks component', () => {
    it('should render correctly', () => {
        const component = shallow(<SignedOutLinks />);
        expect(component).toMatchSnapshot();
    });
});
