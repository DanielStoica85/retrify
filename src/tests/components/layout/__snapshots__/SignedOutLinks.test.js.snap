// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignedInLinks component should render correctly 1`] = `
<Fragment>
  <NavItem
    tag="li"
  >
    <NavLink
      href="/register"
      tag="a"
    >
      Sign Up
    </NavLink>
  </NavItem>
  <NavItem
    tag="li"
  >
    <NavLink
      href="/login"
      tag="a"
    >
      Login
    </NavLink>
  </NavItem>
</Fragment>
`;
